assets/js/app.js

```diff
@@ -40,21 +40,86 @@ liveSocket.connect()
 window.liveSocket = liveSocket
 
 
-/* Message list code */
-const ul = document.getElementById('msg-list');    // list of messages.
-const name = document.getElementById('name');      // name of message sender
-const msg = document.getElementById('msg');        // message input field
-const send = document.getElementById('send');      // send button
+/* INITIAL SETUP OF VARIABLES AND JOINING CHANNEL -------------- */
+const ul = document.getElementById('msg-list');                       // list of messages.
+const name = document.getElementById('name');                         // name of message sender
+const msg = document.getElementById('msg');                           // message input field
+const send = document.getElementById('send');                         // send button
+const usersListMobile = document.getElementById('users_online-list-mobile');      // online users list mobile
```

Comment:
Why is there a difference between who is online on mobile vs. desktop? 🤷‍♂️

Reply:
This is detailed in the [`presence.md` readme file](https://github.com/dwyl/phoenix-chat-example/blob/add_presence-%2314/presence.md#31-change-html-in-view-file).

For the UI to work, I had to create two lists that are in different places within the page. I tried to get just one responsive one but I couldn't in order to not break the rest of the page.